src/RemoteLRS.php

```diff
@@ -901,6 +903,29 @@ public function deleteActivityProfile($activity, $id) {
         return $response;
     }
 
+    public function retrieveActivity($activityid)
+    {
+        $response = $this->sendRequest(
+            'GET',
+            'activities',
+            array(
+                'params' => array(
+                    'activityId' => $activityid,
+                ),
+                'ignore404' => true,
```

Comment:
Given that the LRS should *never* respond with a 404 I don't think we should ignore this for this method.